orchestra/conducting.py

```diff
@@ -69,11 +69,13 @@ def __init__(self, spec, **kwargs):
         self._workflow_state = None
         self._graph = None
         self._flow = None
-        self._inputs = kwargs
+        self._inputs = inputs or {}
+        self._context = context
```

Comment:
Change to `context or {}`? Can we rename `self._context` to `self._init_context` since the actual runtime context is maintained in flow?